src/BannerViewManager.js

```diff
@@ -32,6 +32,7 @@ const getSizeForType = (type: AdType) => sizeForType[type] || sizeForType.standa
 type BannerViewProps = {
   type: AdType,
   placementId: string,
+  style: { [key: string]: any },
```

Comment:
I made a mistake it should be `number|{[key: string]: number|string}`.
A number is for styles which were returned by `Stylesheet.create(...)` method. You can check that method [here](https://github.com/facebook/react-native/blob/master/Libraries/StyleSheet/StyleSheet.js#L197)